meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-xen.dts

```diff
@@ -66,6 +66,10 @@
 		 /* non-prefetchable memory */
 		 0x82000000 0 0x30000000 0 0x30000000 0 0x10000000>;
 	msi-map = <0x0 &gic_its 0x0 0x1000>;
+	interrupt-map = <0 0 0 1 &gic 0 0 GIC_SPI 416 IRQ_TYPE_LEVEL_HIGH
```

Comment:
Patch is named "Fix interrupt-map node", but it adds the node, not fixes it.

Reply:
Well, it fixes the existing entry in the relevant dtsi file by replacing it here

Reply:
`This is fixed by overriding the interrupt-map property from r8a779f0.dtsi.`
added to the commit message